gui-apps/qtgreet-1.0.0.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 2021 Aisha Tammy
```

Comment:
Could you also add your name to the copyright header (if you are comfortable with that)?
It's easier to keep track and give proper credit to you :-)